src/components/PhoneNumberInput/index.js

```diff
@@ -16,19 +21,31 @@ const FlagComponent = ({ countryCode, flagsPath }) => (
   />
 );
 
-const PhoneNumberInput = ({ translate, clearErrors, actions = {}}) => {
+const PhoneNumberInput = ({ translate, clearErrors, actions = {}, smsNumberCountryCode}) => {
 
   const onChange = (number) => {
     clearErrors()
     const valid = isValidPhoneNumber(number)
     actions.setMobileNumber({number, valid})
   }
 
+  const validCountryCode = () => {
+    const isCountryCodeInCountriesList = includes(countries.flat(), lowerCase(smsNumberCountryCode))
+    const isValid = smsNumberCountryCode.length === 2 && isCountryCodeInCountriesList
+    if (!isValid) { console.warn("Invalid ISO Country Code") }
+    return isValid
+  }
+
+  const countryCode = () => {
+    if (!smsNumberCountryCode) return 'GB'
+    return validCountryCode() ? upperCase(smsNumberCountryCode) : 'GB'
```

Comment:
formatting (or normalisation) of the init values should be done at init time as well, if possible. In this case it does seem possible.